m2cgen/assemblers/fallback_expressions.py

```diff
@@ -39,7 +41,7 @@ def sqrt(expr, to_reuse=False):
 
 def exp(expr, to_reuse=False):
     return ast.PowExpr(
-        base_expr=ast.NumVal(2.71828182845904523536028747135),
+        base_expr=ast.NumVal(math.e),
```

Comment:
Refer to https://github.com/BayesWitnesses/m2cgen/pull/220#discussion_r429440112.